Gordon360/Models/AA_ApartmentChoices.cs

```diff
@@ -15,7 +15,7 @@ namespace Gordon360.Models
     public partial class AA_ApartmentChoices
     {
         public int AprtAppID { get; set; }
-        public int Rank { get; set; }
+        public int Ranking { get; set; }
```

Comment:
Out of curiousity, what prompted this naming update? Also, make sure it is updated in any other view models that might involve the apartmentChoice, 

Also, make sure to notify the frontend team so that this name change can be updated in `src/services/housing.js` on the UI, among other places

Reply:
"Rank" is actually a keyword in SQL, so we had to change it.